From 714690b349235dda5cb78008db8d6dd132d82da7 Mon Sep 17 00:00:00 2001
From: Jan Holthuis <jan.holthuis@ruhr-uni-bochum.de>
Date: Fri, 19 Aug 2022 17:55:47 +0200
Subject: [PATCH] [0.32] Load host keys from `/etc/dropbear`

Instead of using the current directory, dropbear should look for the
host keys in `/etc/dropbear`, just like more recent version do.
---
 options.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/options.h b/options.h
index 8912e1b..6476e00 100644
--- a/options.h
+++ b/options.h
@@ -38,8 +38,8 @@
 #define DROPBEAR_PORT 22

 /* Default hostkey paths - these can be specified on the command line */
-#define DSS_PRIV_FILENAME "dropbear_dss_host_key"
-#define RSA_PRIV_FILENAME "dropbear_rsa_host_key"
+#define DSS_PRIV_FILENAME "/etc/dropbear/dropbear_dss_host_key"
+#define RSA_PRIV_FILENAME "/etc/dropbear/dropbear_rsa_host_key"

 /* Disable X11 Forwarding */
 /*#define DISABLE_X11FWD*/
--
2.37.1
